app/controllers/provider/admin/user/access_tokens_controller.rb

```diff
@@ -22,8 +22,16 @@ def create
           create! do |success, _failure|
             success.html do
               flash[:token] = @access_token.id
-              flash[:notice] = 'Access Token was successfully created.'
-              redirect_to(collection_url)
+              redirect_to collection_url, success: t('.success')
+            end
+          end
+        end
+
+        def destroy
```

Comment:
Why is this added?

Reply:
This is to pass our custom flash, otherwise it defaults to notice/alert.